### PR TITLE
port to hatch

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[build-system]
+requires = ["hatchling",
+"hatch-vcs"]
```
Comment (nicoddemus) [inline pyproject.toml:3]:
```suggestion
requires = ["hatchling", "hatch-vcs"]
```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 2",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.5",
```
Comment (nicoddemus) [inline pyproject.toml:26]: 3.5 is failing on CI, I'm not opposed to dropping support for <3.7 in the next release.

Comment (RonnyPfannschmidt) [reply pyproject.toml:26]: i'm about to drop all the old ones

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -97,10 +22,10 @@ jobs:
     - name: Install tox setuptools_scm
       run: |
         python -m pip install --upgrade pip
-        pip install tox setuptools_scm
+        pip install tox setuptools_scm hatch
```
Comment (nicoddemus) [inline .github/workflows/main.yml:25]: Still need setuptools-scm?

**Review by nicoddemus — APPROVED** [review]

File: .github/workflows/main.yml (hunk 3 above)
Comment (RonnyPfannschmidt) [reply .github/workflows/main.yml]: nope, i trimme it down a little

**Pull request merged**